frontend/src/views/CellGuide/components/CellGuideCard/style.ts

```diff
@@ -60,14 +61,15 @@ export const Wrapper = styled.div<CellGuideViewProps>`
   display: flex;
   flex-direction: column;
   align-self: stretch;
-  overflow-x: hidden;
```

Comment:
i'm assuming that this was added for mobile view - so i'm changing the logic to just conditionally hide it depending on if we're in skinny mode or not. in order for the child `CellGuideCardHeader` to stay sticky within its parent container `Wrapper`, `Wrapper` needs to be a scrollable ancestor. removing the overflow in desktop view ensures that `Wrapper` stays scrollable